src/Dapr.Actors/Runtime/Actor.cs

```diff
@@ -300,7 +300,8 @@ protected async Task UnregisterTimerAsync(string timerName)
 
         internal MethodInfo GetMethodInfoUsingReflection(Type actorType, string callback)
         {
-            return actorType.GetMethod(callback, BindingFlags.Public | BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Static);
+            return actorType.GetMethod(callback, BindingFlags.Public | BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Static)
+                ?? actorType.GetMethod($"get_{callback}", BindingFlags.Public | BindingFlags.Instance | BindingFlags.NonPublic | BindingFlags.Static); // F# support
```

Comment:
Can you add a link to https://github.com/fsharp/fsharp/issues/840 or a more descriptive comment here? I worry that the details about *why* this needs to be done this way will be lost to time.